frontend/src/components/TheHeader.vue

```diff
@@ -20,21 +20,21 @@
             <v-row class="align-center">
               <v-col v-if="showMessagingIcon" style="width: 70px">
                 <div>
-                  <v-btn @click="$router.push({ name: 'messaging' })" aria-label="Messages/Notifications" id="mail_box_button" rounded v-if="!isNaN(messageNotificationCount)">
+                  <v-btn v-if="!isNaN(messageNotificationCount)" id="mail_box_button" aria-label="Messages/Notifications" rounded @click="$router.push({ name: 'messaging' })">
                     <v-badge color="error" class="pt-0" :content="messageNotificationCount" bottom right overlap offset-x="8" offset-y="28">
                       <v-icon aria-hidden="false" icon="mdi-email-outline" size="40" color="white" />
                     </v-badge>
                   </v-btn>
-                  <v-skeleton-loader type="chip" class="chip-loader" v-else />
+                  <v-skeleton-loader v-else type="chip" class="chip-loader" />
                 </div>
               </v-col>
-              <v-col class="px-0" style="width: 50px">
-                <v-btn @click="$router.push({ name: 'help' })" aria-label="Help" id="help_button" rounded>
+              <v-col v-if="false" class="px-0" style="width: 50px">
```

Comment:
this col will never be shown? Is that intentional?

Reply:
Yes. We are removing Help temporarily until it's populated.